sort

```diff
@@ -9,3 +9,9 @@ sort -r <file>
 
 # To sort a file randomly:
 sort -R <file>
+
+# To sort a file and store the output in another file:
+sort <inputFile> -o <outputFile>
```

Comment:
```suggestion
# Sort a specific file and store the output in another file:
sort <path/to/input_file> --output=<path/to/output_file>
```

- Optional change: `to` is not mandatory.
- Optional change: `a specific` may sound better.
- Optional change: it's better to use placeholders in the following format: `<path/to/file>`.
- Optional change: it's better to prefer long options because they are easily to memorize.